sstables/sstables.cc

```diff
@@ -970,7 +981,6 @@ void sstable::filesystem_storage::open(sstable& sst, const io_priority_class& pc
         bytes b = bytes(reinterpret_cast<const bytes::value_type *>(value.c_str()), value.size());
         write(sst._version, w, b);
     }
-    w.flush();
```

Comment:
> Flushing the stream in between several writes is not going to
work, because file stream would step on unaligned IO and S3 upload
stream would send completion message to the server and would lose any
subsequent write.

Shouldn't the implementations be changed to throw on flush(), or be a no-op, depending on how we define flush()?

Reply:
Implementation of file and s3 sinks? If yes -- file output sink already has an assertion on alignment. S3 sink completes the multipart upload on flush and it will get assertion (or throw) in case put() happens after flush(), but currently it doesn't